Assignment/Book-Management/src/app/app-routing.module.ts

```diff
@@ -5,7 +5,9 @@ import { NotFoundComponent } from './shared/components/not-found/not-found.compo
 const routes: Routes = [
   {path: "", redirectTo: "book", pathMatch: "full"},
   {path: "book", loadChildren: () => import('./modules/book/book.module').then(m=>m.BookModule)},
+  {path: "category", loadChildren: () => import('./modules/category/category.module').then(m=>m.CategoryModule)},
   {path: "user", loadChildren: () => import('./modules/user/user.module').then(m=>m.UserModule)},
+  {path: "author", loadChildren: () => import('./modules/author/author.module').then(m=>m.AuthorModule)},
```

Comment:
Add space in arrow function. m => m.AuthorModule

Reply:
done